code/modules/jobs/job_types/chief_medical_officer.dm

```diff
@@ -80,7 +80,7 @@
 		/obj/item/gun/syringe,
 		/obj/item/stamp/head/cmo,
 		)
-	skillchips = list(/obj/item/skillchip/entrails_reader)
+	skillchips = list(/obj/item/skillchip/job/chief_medical_officer)
```

Comment:
They should have both skill chips, no?

Reply:
If this was well-received I was hoping we would move in the direction of giving skillchips to other jobs, including medical doctors, that were more specialized. I wanted to keep the entrails skillchip as specific to the coroner.